### PR TITLE
Add libncurses5-dev package, support for Ubuntu 15.10 and 16.10

### PR DESCRIPTION
**Pull request by lekv**
libncurses5-dev is needed to compile GDB and thus the toolchain.

I did not test the checks for Ubuntu 15.10 and 16.04 but don't see why
they would be a problem.

### CODE REVIEW

File: cookbooks/impala/recipes/packages.rb [hunk 1]
```diff
@@ -13,7 +13,8 @@
 when "debian"
   Chef::Log.info('Debian family')
   case
-  when node['platform_version'] == '15.04', node['platform_version'] == '14.04'
+  when node['platform_version'] == "16.04", node['platform_version'] == "15.10",
```
Comment (dtsirogiannis) [inline cookbooks/impala/recipes/packages.rb]: The chef script does not work with 16.04. So, I am not sure it's a good idea to include this here.

**Review by dtsirogiannis — COMMENTED** [review]
I think we need to remove any references to 16.04 as it's not officially supported by the chef script

**Comment by lekv** [comment]
This also lacks the more important addition of libncurses5-dev due to a rebase gone bad. I pushed a new change and will create another PR, since I couldn't figure out how to squash changes in PRs.

**Comment by lekv** [comment]
New PR here: https://github.com/awleblang/impala-setup/pull/8

**Pull request closed without merging**